solo-chains/runtime/starlight/src/genesis_config_presets.rs

```diff
@@ -357,22 +369,12 @@ fn starlight_testnet_genesis(
             "config": runtime_parachains::configuration::HostConfiguration {
                 scheduler_params: SchedulerParams {
                     max_validators_per_core: Some(1),
-                    num_cores: para_ids.len() as u32,
+                    num_cores,
```

Comment:
In https://github.com/moondance-labs/tanssi/pull/664 I'm hardcoding this value to 4 because it makes zombienet testing easier. I don't see a benefit for making this dynamic, but maybe I'm wrong. What do you think @girazoki?

Reply:
@tmpolaczyk Reason why I chose to make it dynamic is because these two parameter together (`num_cores` and parachain percentage) effect the collator assignment and I wanted to make it so that the `num_cores` is dependent upon the parachain percentage chosen and enough cores are available that the other tests that do not care about `num_cores` get expected result. (Without this with some values of `num_cores` @girazoki's tests were failing.)

Reply:
I am ok with making it dynamic, I dont have a strong opinion

Reply:
I think I would leave it as 100% for parchains and maybe leave it at 4 too. For now at least. I understand you need it in the ts tests parth but I think you can change it just for that test. The reason is the vast majority of the tests dont use this value and it would be less disturbant.